index.html

```diff
@@ -119,7 +110,7 @@ <h4>Gracious grad students</h4>
 						<!-- Please keep entries alphabetical by last name. -->
 						<ul>
 							<li>
-								Noura Alomar
```

Comment:
Let's use this page instead for Noura: https://blues.cs.berkeley.edu/people/noura_alomar/